accounts-db/src/accounts_db.rs

```diff
@@ -1343,6 +1343,19 @@ impl StoreAccountsTiming {
     }
 }
 
+#[derive(Default, Debug)]
+struct CleaningInfo {
```

Comment:
This should also include 'purge_older_slots: bool`. That replaces the Vec in the other fn. This can be done in a separate pr as well.

Reply:
Let me add this in a follow-up PR.